org.adempiere.base/src/org/adempiere/util/ReservationLogTracer.java

```diff
@@ -93,6 +94,7 @@ public void trace(BigDecimal originalQty, BigDecimal diffQty) {
 		log.setOldQty(originalQty);
 		log.setNewQty(originalQty.add(diffQty));
 		log.setRecord_ID(Record_ID);
+		log.setAD_Org_ID(MOrg.getIDfromWarehouse(Env.getCtx(), M_Warehouse_ID, trxName));
```

Comment:
m_warehouse_id can't be 0 here (m_storagereservationlog.m_warehouse_id is a not null field), this can be simplify to `MWarehouse.get(ctx, m_warehouse_id, trxName).getAD_Org_ID()`

Reply:
changes applied